### PR TITLE
Run changelog test and save variable at a single line

### PR DESCRIPTION
**Pull request by juliogonzalez**
It seems that using the pipe does not work as I expected in two lines.

### CODE REVIEW

**Comment by MalloZup** [comment]
@juliogonzalez  you need to add white space

**Comment by MalloZup** [comment]
```  sh "${gitarro_changelog_test} ||" +
                sh "${gitarro_changelog_test} || CLRET=\$?;" +
                   "CLRET=\$?;" +
``

  ```  sh "${gitarro_changelog_test} ||" +
                sh "${gitarro_changelog_test} || CLRET=\$?;" +
             ==> whitespace      " CLRET=\$?;" +
``

File: jenkins_pipelines/manager_prs/Jenkinsfile_prs_changelog [hunk 1]
```diff
@@ -46,8 +46,7 @@ pipeline {
         stage('Run changelog test') {
             steps {
                 echo 'Run changelog tests'
-                sh "${gitarro_changelog_test} ||" +
-                   "CLRET=\$?;" +
```
Comment (MalloZup) [inline jenkins_pipelines/manager_prs/Jenkinsfile_prs_changelog]: i mean here after the `+` you would need to add " CLE" ( see whitespace)

**Comment by MalloZup** [comment]
for this small changes feel free to commit directly :+1:

**Comment by juliogonzalez** [comment]
@MalloZup, I need a second pair of eyes, can you please join IRC?

**Comment by MalloZup** [comment]
I am eating

**Comment by MalloZup** [comment]
I will come back in 1h

**Comment by juliogonzalez** [comment]
Sure, no problem :-)

**Pull request merged**